pebblo/app/service/loader/loader_doc_service.py

```diff
@@ -290,6 +320,13 @@ def process_request(self, data):
             else:
                 self.classifier_mode = self.data.get("classifier_mode")
 
+            if not data.get("anonymize_snippets"):
```

Comment:
Do we need this after intializing values of `self.anonymize_snippets` and `self.classifier_mode`?

Reply:
removed